packages/controllers/src/snaps/SnapController.ts

```diff
@@ -726,6 +799,119 @@ export class SnapController extends BaseController<
     }, this._idleTimeCheckInterval) as unknown as number;
   }
 
+  /**
+   * Checks all installed and unblocked snaps against the block list and blocks
+   * snaps as appropriate. See {@link SnapController.blockSnap} for more
+   * information.
+   */
+  async updateBlockedSnaps(): Promise<void> {
+    if (!this._checkSnapBlockList) {
+      return;
+    }
+
+    const blockedSnaps = await this._checkSnapBlockList(
+      Object.values(this.state.snaps).reduce((blockListArg, snap) => {
+        if (!snap.blocked) {
+          blockListArg[snap.id] = snap.version;
```

Comment:
What would be the reasoning for not sending over already blocked snap versions?

Reply:
Shouldn't we send the version so in theory it could be unblocked by updating?

Reply:
Yes we should, this was a hangover from before I added unblocking. Fixed!